src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -184,15 +202,21 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
 
       const data = await tokenRowWalletsInfoStream(walletId, symbol);
 
-      const queryUuid = await data.next();
+      const queryUuid: LiveQueryResult = (await data.next()).value;
```

Comment:
do we need to explicitly use typing here? won't ts figure out type be itself?
lmk if we need that `const queryUuid: LiveQueryResul...`